/uwb_serial:
  ros__parameters:
    usb_port: '/dev/ttyACM0'
    boudrate: 115200
    topic_name: 'uwb/range'
    namespace: ''
    anchors_calib_files:
      - 'calib1.csv'
      - 'calib2.csv'
    compensation: 0 # 0: no compensation, 1: lookup table compensation, 2: linear interpolation